tools/fonts/README.md

```diff
@@ -0,0 +1,63 @@
+## Font Generation Scripts
```

Comment:
This is going to diverge very quickly from the Wiki. I don't have a strong opinion for where this goes, but for consistency, the wiki would be it.

Whatever we decide, can we have one redirect to the other?